src/components/FeatureList.tsx

```diff
@@ -1,60 +1,61 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import {View} from 'react-native';
-import _ from 'underscore';
+import type {StyleProp, ViewStyle} from 'react-native';
 import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
-import stylePropTypes from '@styles/stylePropTypes';
 import variables from '@styles/variables';
+import type {TranslationPaths} from '@src/languages/types';
+import type IconAsset from '@src/types/utils/IconAsset';
 import Button from './Button';
+import type DotLottieAnimation from './LottieAnimations/types';
 import MenuItem from './MenuItem';
-import menuItemPropTypes from './menuItemPropTypes';
 import Section from './Section';
 
-const propTypes = {
+type FeatureListItem = {
+    icon: IconAsset;
+    translationKey: TranslationPaths;
+};
+
+type FeatureListProps = {
     /** The text to display in the title of the section */
-    title: PropTypes.string.isRequired,
+    title: string;
 
     /** The text to display in the subtitle of the section */
-    subtitle: PropTypes.string,
+    subtitle?: string;
 
     /** Text of the call to action button */
-    ctaText: PropTypes.string,
+    ctaText?: string;
 
     /** Accessibility label for the call to action button */
-    ctaAccessibilityLabel: PropTypes.string,
+    ctaAccessibilityLabel?: string;
 
     /** Action to call on cta button press */
-    onCtaPress: PropTypes.func,
+    onCtaPress?: () => void;
 
     /** A list of menuItems representing the feature list. */
-    menuItems: PropTypes.arrayOf(PropTypes.shape({...menuItemPropTypes, translationKey: PropTypes.string})).isRequired,
+    menuItems: FeatureListItem[];
 
     /** The illustration to display in the header. Can be a JSON object representing a Lottie animation. */
-    illustration: PropTypes.shape({
-        file: PropTypes.oneOfType([PropTypes.string, PropTypes.number]).isRequired,
-        w: PropTypes.number.isRequired,
-        h: PropTypes.number.isRequired,
-    }),
+    illustration: DotLottieAnimation;
 
     /** The style passed to the illustration */
-    illustrationStyle: stylePropTypes,
+    illustrationStyle?: StyleProp<ViewStyle>;
 
     /** The background color to apply in the upper half of the screen. */
-    illustrationBackgroundColor: PropTypes.string,
-};
-
-const defaultProps = {
-    ctaText: '',
-    ctaAccessibilityLabel: '',
-    subtitle: '',
-    onCtaPress: () => {},
-    illustration: null,
-    illustrationBackgroundColor: '',
-    illustrationStyle: [],
+    illustrationBackgroundColor?: string;
 };
 
-function FeatureList({title, subtitle, ctaText, ctaAccessibilityLabel, onCtaPress, menuItems, illustration, illustrationStyle, illustrationBackgroundColor}) {
+function FeatureList({
+    title,
+    subtitle = '',
+    ctaText = '',
+    ctaAccessibilityLabel = '',
+    onCtaPress = () => {},
```

Comment:
is it needed? onPress in Button is optional

Reply:
updated